components/gripper.py

```diff
@@ -36,6 +40,10 @@ def get_full_closed(self) -> bool:
             (time.monotonic() - self.close_time) >= Gripper.CLOSE_TIME_THERESHOLD
         ) and (not self.opened)
 
+    @feedback
+    def get_full_open(self) -> bool:
+        return time.monotonic() - self.open_time >= Gripper.OPEN_TIME_THERESHOLD
```

Comment:
I think this will still return true when the claw has been closed

Reply:
I agree with Oliver. This will latch true because the check is only based on time. The easiest fix is to also check if the gripper is still commanded to be in the opened state. have a look at the ```get_full_closed``` function if you are confused by what I am suggesting.